[Feat notification_views] 💡 chore : 실제 api로 전환

LuSiEDa opened the pull request:
## 📄 상세 내용
- [ ] notification_views 목 데이터 제거
- [ ]  실제 api로 전환
- [ ] 그에 맞춰 테스트 파일도 함께 수정


## 📸 스크린샷 (선택)

## 📝 기타 참고 사항

## 🧪 PR Checklist
- [ ] 커밋 메시지 컨벤션에 맞게 작성했습니다.
- [ ] 변경 사항에 대한 테스트를 했습니다.(버그 수정/기능에 대한 테스트).

LuSiEDa committed:
# Commit Convention
# ✨ feat     : 새로운 기능 추가
# 🐛 fix      : 버그 수정
💡 chore    : notification_views 목 데이터 제거, 실제 api로 전환, 그에 맞춰 테스트 파일도 함께 수정
# 경🎨 style    : 코드 포매팅 수정
# 📝 docs     : 문서 수정 (README 등)
# 🚚 build    : 빌드 관련 파일 수정
# ✅ test     : 테스트 코드 추가/변경 (프로덕션 코드 변경 없음)
# ♻️ refactor : 리팩터링 (기능 변화 없음)
# 🚑 hotfix   : 긴급 수정
#
# 아래 1번 문항부터 주석 문구가 빈줄에 주석을 지우고 문항에 대한 내용을 작성하고 커밋을 완료해주세요.
#
# 1. 아래 형식에 맞춰 커밋 메시지 타이틀을 작성하세요:
# <이모지> <타입>: <간결한 커밋 메시지 요약>
#
# 예시:
# ✨ feat: 사용자 로그인 기능 추가
# 🐛 fix: 댓글 생성 시 발생하는 NullPointerException 수정
# 💡 chore: 불필요한 로그 제거 및 변수명 수정
# 🎨 style: black, isort 코드 포매터 실행
# 📝 docs: README에 프로젝트 설명 추가
# 🚚 build: Dockerfile 수정하여 실행 오류 해결
# ✅ test: 게시글 API 단위 테스트 추가
# ♻️ refactor: 중복 코드 제거 및 함수 분리
# 🚑 hotfix: 프로덕션 장애 수정 - 잘못된 URL 패턴 수정

# 2. 변경 또는 추가사항을 아래에 간략하게 작성하세요 ( 필수 )
#
# 본문 내용은 어떻게 변경했는지 보다 무엇을 변경했는지 또는 왜 변경했는지를 설명합니다.

# 3. 이슈가 있다면 아래에 연결하세요 ( 선택 )
#
# 예시
# 관련 이슈: #123

SWWS97 commented on apps/notification/views/notification_views.py:
```diff
@@ -167,40 +46,31 @@ class NotificationListAPIView(APIView):
-        paginator = MockCursorPagination()
-        paginated_qs = paginator.paginate_queryset(qs, request)
+        paginator = self.pagination_class()
+        paginated_qs: list[Notification] = paginator.paginate_queryset(qs, request)  # type: ignore
```
이부분 타입힌트 제거 하시고 paginated_qs = paginator.paginate_queryset(qs, request) 이렇게 사용 해보실래요?

LuSiEDa committed:
chore    : 71번째 줄 타입힌트, 타입 이그노어 제거
# 경🎨 style    : 코드 포매팅 수정
# 📝 docs     : 문서 수정 (README 등)
# 🚚 build    : 빌드 관련 파일 수정
# ✅ test     : 테스트 코드 추가/변경 (프로덕션 코드 변경 없음)
# ♻️ refactor : 리팩터링 (기능 변화 없음)
# 🚑 hotfix   : 긴급 수정
#
# 아래 1번 문항부터 주석 문구가 빈줄에 주석을 지우고 문항에 대한 내용을 작성하고 커밋을 완료해주세요.
#
# 1. 아래 형식에 맞춰 커밋 메시지 타이틀을 작성하세요:
# <이모지> <타입>: <간결한 커밋 메시지 요약>
#
# 예시:
# ✨ feat: 사용자 로그인 기능 추가
# 🐛 fix: 댓글 생성 시 발생하는 NullPointerException 수정
# 💡 chore: 불필요한 로그 제거 및 변수명 수정
# 🎨 style: black, isort 코드 포매터 실행
# 📝 docs: README에 프로젝트 설명 추가
# 🚚 build: Dockerfile 수정하여 실행 오류 해결
# ✅ test: 게시글 API 단위 테스트 추가
# ♻️ refactor: 중복 코드 제거 및 함수 분리
# 🚑 hotfix: 프로덕션 장애 수정 - 잘못된 URL 패턴 수정

# 2. 변경 또는 추가사항을 아래에 간략하게 작성하세요 ( 필수 )
#
# 본문 내용은 어떻게 변경했는지 보다 무엇을 변경했는지 또는 왜 변경했는지를 설명합니다.

# 3. 이슈가 있다면 아래에 연결하세요 ( 선택 )
#
# 예시
# 관련 이슈: #123
